Team6220_2021/src/main/java/org/firstinspires/ftc/team6220_2021/RedRightAuto.java

```diff
@@ -0,0 +1,143 @@
+package org.firstinspires.ftc.team6220_2021;
+
+import android.annotation.SuppressLint;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+
+import org.firstinspires.ftc.robotcore.external.ClassFactory;
+import org.firstinspires.ftc.robotcore.external.hardware.camera.WebcamName;
+import org.firstinspires.ftc.robotcore.external.navigation.VuforiaLocalizer;
+import org.firstinspires.ftc.robotcore.external.tfod.Recognition;
+import org.firstinspires.ftc.robotcore.external.tfod.TFObjectDetector;
+import org.firstinspires.ftc.team6220_2021.ResourceClasses.Constants;
+
+import java.util.List;
+
+@Autonomous(name = "RedRightAuto", group = "Competition")
+public class RedRightAuto extends MasterAutonomous {
```

Comment:
A lot of these seem very similar. You should refactor to reuse as much code as possible.